### PR TITLE
Creation of a function to get all react component

### PR DESCRIPTION
**Pull request by epoundor**

### CODE REVIEW

**Review by YacineSteeve — APPROVED** [review]
Looks good to me

File: apps/frontend/src/main.tsx [hunk 1]
```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
 import './index.css';
-
+import { Button } from '@react-site-editor/ui';
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
     <React.StrictMode>
+        <Button
```
Comment (eliotfgn) [inline apps/frontend/src/main.tsx:8]: App is the only component in main

File: packages/ui/src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface Props {
```
Comment (eliotfgn) [inline packages/ui/src/components/Button.tsx:3]:
```suggestion
interface ButtonProps {
```

File: packages/ui/src/components/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    text: string;
+    onClick: () => void;
+    style?: React.CSSProperties;
+}
+
+const Button: React.FC<Props> = ({ text, onClick, style }) => {
```
Comment (eliotfgn) [inline packages/ui/src/components/Button.tsx:9]: Why React.FC ??

**Review by eliotfgn — CHANGES_REQUESTED** [review]

**Review by eliotfgn — APPROVED** [review]
Good !

**Pull request merged**